apps/kennisbank-dashboard/package.json

```diff
@@ -17,17 +16,23 @@
   },
   "dependencies": {
     "@frameless/strapi-tiptap-editor": "0.0.0-semantically-released",
-    "@strapi/plugin-graphql": "4.14.4",
-    "@strapi/plugin-i18n": "4.14.4",
-    "@strapi/plugin-users-permissions": "4.14.4",
-    "@strapi/strapi": "4.14.4",
+    "@strapi/plugin-graphql": "4.14.6",
+    "@strapi/plugin-i18n": "4.14.6",
+    "@strapi/plugin-users-permissions": "4.14.6",
+    "@strapi/strapi": "4.14.6",
     "better-sqlite3": "8.3.0",
     "pg": "8.11.3",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-is": "18.2.0",
+    "react-router-dom": "5.2.0",
```

Comment:
I think that the dependencies mentioned are not required until the version `@strapi/strapi: 4.14.6`.